frontend/src/components/userPage/DisplayUserRidesharings.jsx

```diff
@@ -0,0 +1,45 @@
+const DisplayUserRidesharings = ({ user, userRideshares }) => {
+    const renderRidesharings = () => {
+        return userRideshares.map((rideshare) => (
+            <div
+                className="mx-[1rem] my-[1rem] border-2 border-black rounded h-fit p-[.5rem]"
+                key={rideshare.id}
+            >
+                <h3 className="text-white font-sans font-bold bg-blue-950 rounded overflow-hidden w-fit h-fit p-[.4rem] mb-[.5rem]">
+                    {rideshare.id}
+                </h3>
+                <p className="bg-white h-fit p-[.2rem] break-all">
+                    ⊛ {rideshare.seatsAvailable} seats available
+                </p>
+                <p className="bg-white h-fit p-[.2rem] break-all">
+                    ⊛ {rideshare.departingTime.substring(0, 10)}
+                </p>
+                <p className="bg-white h-fit p-[.2rem] break-all">
+                    ⊛ {rideshare.departingTime.substring(11, 16)}
+                </p>
```

Comment:
Recurring comments: https://github.com/CristobalMe/Event_Finder/pull/37#discussion_r1681422782

Reply:
I just want to print it here. 
Should I parse it here too?